public/docs/ts/latest/guide/animations.jade

```diff
@@ -292,60 +286,59 @@ figure
 :marked
-  With animation *keyframes* we can go beyond a simple transition between two
-  sets of styles to a more intricate animation that goes through one or more
-  intermediate styles in between.
+  Animation *keyframes* go beyond a simple transition to a more intricate animation
+  that goes through one or more intermediate styles when transitioning between two sets of styles.
 
-  For each keyframe, we can specify an *offset* that defines at which point
-  in the animation that keyframe applies. The offset is a number between zero,
+  For each keyframe, you specify an *offset* that defines at which point
+  in the animation keyframe applies. The offset is a number between zero,
   which marks the beginning of the animation, and one, which marks the end.
 
-  In this example we add some "bounce" to our enter and leave animations with
+  This example adds some "bounce" to the enter and leave animations with
   keyframes:
 
 +makeExample('animations/ts/app/hero-list-multistep.component.ts', 'animationdef')(format=".")
 
 :marked
   Note that the offsets are *not* defined in terms of absolute time. They are relative
-  measures from 0 to 1. The final timeline of the animation will based on the combination
+  measures from zero to one. The final timeline of the animation is based on the combination
   of keyframe offsets, duration, delay, and easing.
 
-  Defining offsets for keyframes is optional. If we omit them, offsets with even
+  Defining offsets for keyframes is optional. If you omit them, offsets with even
   spacing are automatically assigned. For example, three keyframes without predefined
-  offsets will receive offsets `0`, `0.5`, and `1`.
+  offsets receive offsets `0`, `0.5`, and `1`.
 
 :marked
-  ## Parallel Animation Groups
+  ## Parallel animation groups
 figure
   img(src="/resources/images/devguide/animations/animation_groups.gif" alt="Parallel animations with different timings, implemented with groups" align="right" style="width:220px;margin-left:20px" )
 :marked
-  We've already seen how we can animate multiple style properties at the same time:
-  Just put all of them into the same `style()` definition!
+  You've seen how you to animate multiple style properties at the same time:
+  just put all of them into the same `style()` definition.
 
-  But we may also want to configure different *timings* for animations that happen
-  in parallel. For example, we may want to animate two CSS properties but use a
+  But you may also want to configure different *timings* for animations that happen
+  in parallel. For example, you may want to animate two CSS properties but use a
   different easing function for each one.
 
-  For this we can use animation *groups*. In this example we use groups both on
-  enter and leave so that we can use two different timing configurations. Both
+  For this you can use animation *groups*. In this example, using groups both on
+  enter and leave allow for two different timing configurations. Both
```

Comment:
allow -> allows
(since it's "using", not "groups", that's the subject of this verb)